### ISSUE TITLE
Tag and IID always None if UseNumeric is set

### ISSUE THREAD

**Issue by spacefreak86** [net-snmp/net-snmp#419]
I use the Python library to retrieve values from SNMP servers, here is the example code:

```
#!/usr/bin/env python
import netsnmp

var = netsnmp.Varbind('hrStorageDescr')
varlist = netsnmp.VarList(var)

netsnmp.snmpwalk(
    varlist,
    DestHost='localhost',
    Version=2,
    Community='public',
    UseNumeric=1)

for var in varlist:
    print(var.tag, var.iid, var.val, var.type)`
```

Until **net-snmp version 5.9**, this is working as it should, the output looks like this:
```
.1.3.6.1.2.1.25.2.3.1.3 1 Physical memory OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 3 Virtual memory OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 6 Memory buffers OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 7 Cached memory OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 8 Shared memory OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 10 Swap space OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 31 / OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 34 /run OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 40 /sys/fs/cgroup OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 58 /dev/shm OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 60 /mnt/data OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 61 /var/lib/mysql OCTETSTR
.1.3.6.1.2.1.25.2.3.1.3 63 /run/user/0 OCTETSTR
```

With **net-snmp version 5.9.1**, tag and iid are always None:
```
None None Physical memory OCTETSTR
None None Virtual memory OCTETSTR
None None Memory buffers OCTETSTR
None None Cached memory OCTETSTR
None None Shared memory OCTETSTR
None None Swap space OCTETSTR
None None Available memory OCTETSTR
None None / OCTETSTR
None None /dev/shm OCTETSTR
None None /run OCTETSTR
None None /sys/fs/cgroup OCTETSTR
None None /tmp OCTETSTR
None None /run/user/1000 OCTETSTR
```

It seems that it has something to do with the UseNumeric parameter. If it is set to zero, the output looks like this with both versions of net-snmp:
```
hrStorageDescr 1 Physical memory OCTETSTR
hrStorageDescr 3 Virtual memory OCTETSTR
hrStorageDescr 6 Memory buffers OCTETSTR
hrStorageDescr 7 Cached memory OCTETSTR
hrStorageDescr 8 Shared memory OCTETSTR
hrStorageDescr 10 Swap space OCTETSTR
hrStorageDescr 11 Available memory OCTETSTR
hrStorageDescr 31 / OCTETSTR
hrStorageDescr 37 /dev/shm OCTETSTR
hrStorageDescr 39 /run OCTETSTR
hrStorageDescr 40 /sys/fs/cgroup OCTETSTR
hrStorageDescr 63 /tmp OCTETSTR
hrStorageDescr 64 /run/user/1000 OCTETSTR
```

Unfortunately, I need the numeric output of the tags and iids, so this is no solution for me. I have already tried to track down the issue within the C source, but without success so far.

**Comment by THERAT84** [comment]
I ran into the same issue. A fix would be really nice

**Comment by R4ngered** [comment]
Hi @fenner 
Could you please take a look at this issue. The same problem happened to me when using the library.
Thanks and kind regards

**Comment by bvanassche** [comment]
Please retest with the latest version of the V5-9-patches branch. This should already have been fixed by commit 8c1dad23301692799749d75a3c039b8ae7c07f8e.
